packages/core/src/components/dialog/_dialog.scss

```diff
@@ -135,23 +136,3 @@ $dialog-padding: $pt-grid-size * 1.5 !default;
     border-bottom: 1px solid $pt-high-contrast-mode-border-color;
   }
 }
-
-.#{$ns}-dialog-body {
-  flex: 1 1 auto;
-  line-height: $pt-grid-size * 1.8;
```

Comment:
unfortunately we need to keep this style too, it's too risky to change, and it's part of our API contract not to break styles on public class names like this.

Reply:
actually, I take this back -- this is the default line height, and it's applied by the `base-typography()` Sass mixin on `body` already, so it's probably ok to remove this style :+1: